src/utils/github_updater.rs

```diff
@@ -189,6 +187,31 @@ impl GithubUpdater {
         Ok(())
     }
 
+    fn repository_name(url: &str) -> Option<RepositoryName> {
+        static RE: Lazy<Regex> = Lazy::new(|| {
+            Regex::new(r"https?://(www.)?github\.com/(?P<owner>[\w\._-]+)/(?P<repo>[\w\._-]+)")
+                .unwrap()
+        });
+
+        match RE.captures(url) {
+            Some(cap) => {
+                let owner = cap.name("owner").expect("missing group 'owner'").as_str();
+                let repo = cap.name("repo").expect("missing group 'repo'").as_str();
+                Some(RepositoryName {
+                    owner,
+                    repo: repo.strip_suffix(".git").unwrap_or(repo),
+                })
+            }
+            None => None,
+        }
+    }
+
+    fn name() -> &'static str {
+        "Github"
+    }
```

Comment:
Why have this if you never use it? I think it makes more sense to make this `"github"` so you can avoid hard coding the same string everywhere, or just remove it.

Reply:
It's used in `RepositoryStatsUpdater`. I use it in the log messages.